pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix

```diff
@@ -46,23 +47,41 @@ buildPythonPackage rec {
     psycopg-pool
   ];
 
-  pythonRelaxDeps = [ "psycopg-pool" ];
+  pythonRelaxDeps = [
+    "langgraph-checkpoint"
+    "psycopg-pool"
+  ];
 
   doCheck = !(stdenvNoCC.hostPlatform.isDarwin);
 
-  pythonImportsCheck = [ "langgraph.checkpoint.postgres" ];
-
   nativeCheckInputs = [
-    postgresql
-    postgresqlTestHook
     pytest-asyncio
     pytestCheckHook
+    postgresqlTestHook
+    (postgresql.withPackages (p: [ p.pgvector ]))
   ];
 
-  passthru = {
-    updateScript = langgraph-sdk.updateScript;
+  env = {
+    postgresqlTestUserOptions = "LOGIN SUPERUSER";
```

Comment:
I'm unsure about this, shouldn't this be only made during the tests ?

Reply:
This variable is only used by the postgresqlCheckHook and this is a fairly common way to use it. I could set it in `preCheck` if you like.

Reply:
I think it would be a better idea.

WDYT @GaetanLepage ?

Reply:
Give me a sec, I'll do it.

Reply:
Moved into `preCheck` (which clarifies its use better, thank you).

Reply:
Good for me ! `preCheck` makes sense indeed.